packages/config/src/api/site_info.js

```diff
@@ -15,26 +17,30 @@ export const getSiteInfo = async function ({
   siteId,
   mode,
   siteFeatureFlagPrefix,
+  featureFlags = {},
   testOpts: { env: testEnv = true } = {},
 }) {
   if (api === undefined || mode === 'buildbot' || !testEnv) {
     const siteInfo = siteId === undefined ? {} : { id: siteId }
     return { siteInfo, accounts: [], addons: [] }
   }
+  const fetchIntegrations = featureFlags.buildbot_fetch_integrations
+
+  const promises = [getSite(api, siteId, siteFeatureFlagPrefix), getAccounts(api), getAddons(api, siteId)]
+
+  if (fetchIntegrations) {
+    promises.push(getIntegrations({ api, owner: 'site', ownerId: siteId }))
```

Comment:
You're calling the method with a property called `owner` but the method seems to expect `ownerType`?

Reply:
where would we be without eduardo

I am _so_ spoilt by type checking it's not even funny